### ISSUE TITLE
openwatcom.org is dark (no http or ping service)

### ISSUE THREAD

**Issue by ideafarm** [open-watcom/open-watcom-v2#173]
Does anyone know why?  Does anyone have regular contact with Perforce?  (Perforce administers the domain and provides the server.)  Did they pull the plug because they are feeling unappreciated?  Can someone go find out and smooth their ruffled feathers?

**Comment by ideafarm** [comment]
Re: http://openwatcom.ideafarm.com/

I have created the above subdomain to provisionally provide basic information about OW.  If openwatcom.org is dark because Perforce pulled the plug intentionally, it might take some time to resolve whatever issues caused that.  In the meantime, I would like someone to help me gather a URL and an introductory paragraph for each active OW repository and group.

I intentionally have created the list yet, because I want to avoid any appearance that the subdomain that I just created favors this or any particular group or repository.  As soon as possible, I'd like to create the list to point at least to the 1.9 contributors and to this (Jiri's) repo.  Once that is done, I'd like people to help me fill it out so that it lists all active repositories and groups.

**Comment by pchapin** [comment]
I wouldn't panic about www.openwatcom.org. It is most likely a technical glitch on the server side. These kinds of things have happened before.

**Comment by ideafarm** [comment]
It's still dark, and it's been 15 hours since I happened to try to load it.  Who knows how much longer it has been dark.

Sites such as IDEAFARM.COM, served by GoDaddy.com, never go dark for more than a few minutes, if at all.  A Rackspace.com CDN site would theoretically never ever go dark.

IMO, the casual attitude that you express is another indication that no one is looking at OW from an end-user POV or from the POV of preserving and growing OW's market penetration.  As an economist, I find this interesting in that it might be a general flaw in the open source production model.  Coders never appreciate the marketing department, but every CEO knows that both functions need to be alive and well for the product to succeed and to continue to succeed.

I don't see the OW source code as a plaything that exists for the benefit of young, inexperienced amateur coders.  Although I'm as much of a coder as anyone here, I am at this moment looking at the current OW situation as a CEO or a marketing manager would look at it.

If openwatcom.org goes dark occasionally, as you suggest, then perhaps it is time to work with the 1.9 group to persuade them that the site should be neutral with respect to the various groups and repositories, and that the content should be controlled by the overall OW contributor community, not by any particular group.  To put it bluntly, any notion that the 1.9 group is the "official group", with a somehow elevated status, should be brought out into the open, discussed, and put to rest.  The OW user community is best served, and the value and effectiveness of the OW trademark is best preserved, by using the domain and the trademark in a way that is neutral so that all groups and repositories are presented as parts of a single community of people who both compete and cooperate and who are having fun.

**Comment by ideafarm** [comment]
@pchapin openwatcom.org is still dark.  I am going to contact Perforce.  I don't know anyone there.  If you know anyone there or know who has been administering the server and the domain registrations for us, please reply with contact information, or send me email: ideafarmcity@gmail.com .  (I just posted an exact duplicate of this in the contributors group.)

**Comment by ideafarm** [comment]
I just used Perforce's contact form to send them this:

RE: openwatcom.org openwatcom.com domains

Perforce has supported, in several ways, the Open Watcom open source project since the Watcom C/C++ and Fortran compiler and tools were released into open source circa 2000.  Perforce has provided a server and a free Perforce license, and has administered the two domains listed above.

A few days ago, openwatcom.org went dark.  Please put me into email contact with whoever at Perforce has been supporting the Open Watcom contributor and user communities.

I am an independent software developer, and have a strategic interest in, and commitment to, ensuring the survival and development of the OW communities and open source product.

Is openwatcom.org dark due to a technical problem, or due to a management decision at Perforce?  If the latter, I would like to discuss whatever issues have come up and what I can do to assist in resolving them.

**Comment by pchapin** [comment]
The last I knew Perforce doesn't run the server, they just pay for it. Probably it would be best to contact the service provider. Marty Stanquist is the current maintainer, last I knew, so he should have contact information. I might have some very old contact information but it is likely out of date and also buried in some archived backup that I'd rather not dig out right now.

**Comment by ideafarm** [comment]
I just received this news from Mr. Stanquist:  "The server went down due to a technical issue. Perforce is aware of this and is working on it."  Let's keep this issue open until it comes back up.

**Comment by jmalak** [comment]
It is more then week down and nobody is interested to get it up, it looks like it is out of OW community control.
For me is unknown what really happened and who and what should do to be working again.
No info about OW official infrastructure, if server is maintained by OW guys and which one or if it is maintained by service provider (virtual or dedicated server) or directly by Perforce company etc.

It is reason why I don't want to use any private resources and services for OW2 fork core, to be really open and sharable (don't depend on one person, hardware etc.).

**Comment by rdos314** [comment]
It looks like it is Perforce that has registered the domain. From whois:
Registrant Contact Information:
    Name: Perforce Software
    Organization: Perforce Software
    City: Alameda
    State: CA
    Zip: 94501
    Country: US
    Phone: +1.5108647400
    Email: seivald@PERFORCE.COM

On the website it lists:
Owner Name SciTech Software, Inc.
Email KendallJBennett@yahoo.com
Address 505 Wall St, Chico, CA  	95928,US

**Comment by pchapin** [comment]
The server was compromised and shutdown to block attacks being made by it against other systems in the providers domain. They are in the process of rebuilding a new system for openwatcom.org with updated software across the board. Once complete, the existing resources (Perforce depot, wiki, files, etc) will be transferred to the new system and the new system will be brought back online.

This information was sent to a small collection of people by Marty. I guess I was on the list because I was the project maintainer in the past.

**Comment by jmalak** [comment]
Peter thanks for your info.
